hvac/api/secrets_engines/transit.py

```diff
@@ -805,6 +808,14 @@ def sign_data(
         :type salt_length: str | unicode
         :param mount_point: The "path" the method/backend was mounted on.
         :type mount_point: str | unicode
+        :param batch_input: Specifies a list of items for processing.
+            This parameter is mutually exclusive with the ``hash_input`` parameter, but one of them must be supplied. 
+            If both are set, or neither are set, an exception will be raised.
+            Responses are returned in the ``batch_results`` array component of the ``data`` element of the response.
+            Any batch output will preserve the order of the batch input.
+            If the input data value of an item is invalid, the corresponding item in the ``batch_results``
+            will have the key ``error`` with a value describing the error.
```

Comment:
some more re-ordering to put description first and align with the other param.
```suggestion
        :param batch_input: Specifies a list of items for processing.
            Any batch output will preserve the order of the batch input.
            If the input data value of an item is invalid, the corresponding item in the ``batch_results``
            will have the key ``error`` with a value describing the error.
            This parameter is mutually exclusive with the ``hash_input`` parameter, but one of them must be supplied. 
            If both are set, or neither are set, an exception will be raised.
            Responses are returned in the ``batch_results`` array component of the ``data`` element of the response.
```